datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -720,5 +720,76 @@ CREATE TABLE dcsa_im_v3_0.vessel_schedule_terminal_visits (
     created_date_time timestamp with time zone NOT NULL DEFAULT now()
 );
 
+--- Supporting tables for Electronic Endorcement chain PoC
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.addressbookentry CASCADE;
+CREATE TABLE dcsa_im_v3_0.addressbookentry (
+	id int4 NOT NULL GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+	ebl_platform varchar(255) NULL,
+	"name" text NULL,
+	public_key text NULL,
+	thumbprint varchar(255) NULL
+);
+
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.billoflading CASCADE;
+CREATE TABLE dcsa_im_v3_0.billoflading (

```

Comment:
Is this table needed? I would assume that the transport_document_json would be enough. With this table we need to support BoL structures (which might change...)...